api/src/ApplicationError.ts

```diff
@@ -0,0 +1,41 @@
+import { ErrorCode, ERRORS, ErrorTypes } from "./errors";
+
+type ApplicationErrorOptions = {
+  isOperational: boolean;
+  exposeToClient: boolean;
+};
+
+const defaultOptions: ApplicationErrorOptions = {
+  isOperational: true,
+  exposeToClient: true,
```

Comment:
Exposing to client is relatively safe in this application. 

Typically you don't want this where there's a frontend involved, since they won't know how to resolve this sort of error.